sdk/tests/conformance/extensions/oes-vertex-array-object.html

```diff
@@ -546,22 +547,102 @@
     // Render with the deleted buffers. As they are referenced by VAOs they
     // must still be around.
     for (var ii = 0; ii < colors.length; ++ii) {
-      var color = colors[ii];
-      ext.bindVertexArrayOES(vaos[ii]);
-      gl.drawElements(gl.TRIANGLES, 6, gl.UNSIGNED_BYTE, 0);
-      wtu.checkCanvas(gl, color, "should be " + color);
+        var color = colors[ii];
+        ext.bindVertexArrayOES(vaos[ii]);
+        gl.drawElements(gl.TRIANGLES, 6, gl.UNSIGNED_BYTE, 0);
+        wtu.checkCanvas(gl, color, "should be " + color);
     }
 
     // Clean up.
     for (var ii = 0; ii < colorBuffers.length; ++ii) {
-      ext.deleteVertexArrayOES(vaos[ii]);
+        ext.deleteVertexArrayOES(vaos[ii]);
     }
 
     for (var ii = 0; ii < colorBuffers.length; ++ii) {
-      // The buffers should no longer be valid now that the VAOs are deleted
-      if(gl.isBuffer(colorBuffers[ii])) {
-        testFailed("buffer not properly cleaned up after VAO deletion");
-      }
+        // The buffers should no longer be valid now that the VAOs are deleted
+        if(gl.isBuffer(colorBuffers[ii])) {
+            testFailed("buffer not properly cleaned up after VAO deletion");
+        }
+    }
+}
+
+function runBoundDeleteTests() {
```

Comment:
Only this runBoundDeleteTests is a new test which is backported from https://github.com/KhronosGroup/WebGL/blob/master/sdk/tests/conformance2/vertex_arrays/vertex-array-object.html#L514. Other part in this file is just formatting.